docs/articles/expensify-classic/billing-and-subscriptions/Free-Trial.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Free Trial
 description: Free Trial
+redirect_from: articles/split-bills/workspaces/The-Free-Plan/
```

Comment:
I could be wrong, but I don't think `billing-and-subscriptions/Free-Trial` is actually the same doc page as The Free Plan. cc @maddylewis